Foundation/Data.swift

```diff
@@ -790,7 +790,7 @@ public final class _DataStorage {
     }
 }
 
-internal class _NSSwiftData : NSData {
+public class _NSSwiftData : NSData {
```

Comment:
If we mark this public then others will come to depend upon it, making us unable to improve it in the future.

Supposedly the 'testable' feature is what we need for this, but I don't know what the support for testable is like on Linux.